test/data/eval/success/golden/example10.json

```diff
@@ -1,22 +1,22 @@
 [
     {
-        "name": "Shepard Mccall",
-        "id": 0
+        "id": 0,
+        "name": "Shepard Mccall"
```

Comment:
Why does the order change for this?

Reply:
I'm not really sure TBH. The order comes from serializing hashmaps as json objects. In one of the later PRs I replace the hashmaps with assoc lists (which should be insert order hash maps once I write a Lift instance for it) so this goes away.